### DISCUSSION TITLE
Restore failing, even though Log files say "successful run"

### DISCUSSION THREAD

**Discussion by embreeb** [belowaverage-org/SuperGrate#121] [Q&A]
I have just started using SuperGrate, and I noticed it is not restoring to the Destination 100% every time. The log files show it ran successfully, and no errors that would tell me why it failed the restore. It missed Desktop, Documents and Pictures, but did transfer printers and Outlook Signatures.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Logs**
```
Please paste logs here in a code block.

**IMPORTANT**

To retrieve the logs, click "File", the select "Save log..." Copy the contents of the log file and paste it here.
Refrain from copying and pasting the log from the console since it does not output verbose information.

```

**Additional context**

**Comment by krisdb2009** [comment]
Can you share the log file?
